CODEOWNERS

```
@@ -5,4 +5,4 @@
 # For more details, read the following article on GitHub: https://help.github.com/articles/about-codeowners/.
 
 # The default owners are automatically added as reviewers when you open a pull request unless different owners are specified in the file.
-* @fmvilas @derberg @asyncapi-bot-eve
+* @fmvilas @derberg @magicmatatjahu @asyncapi-bot-eve
```

I think this should go in a separate PR 🙏 

```suggestion
* @fmvilas @derberg @asyncapi-bot-eve
```

---

Ok :)